airflow/providers/fab/auth_manager/security_manager/override.py

```diff
@@ -1102,81 +1103,103 @@ def sync_perm_for_dag(
 
         :param dag_id: the ID of the DAG whose permissions should be updated
         :param access_control: a dict where each key is a role name and
-            each value is a set() of action names (e.g.,
+            each value is a set() of action names in case of DAGs resource (e.g.,
             {'can_read'}
+            or the value can be a dict where each key is a resource name and
+            each value is a set() of action names (e.g., {'DAG Runs': {'can_read'}})
```

Comment:
Try to clarify the description, it's quite hard to read (same goes for similar docstrings)

Reply:
Tried to make it more clear in https://github.com/apache/airflow/pull/40703/commits/c7ebe69d6fe41e37262ace566d70d489a895fc75 . Please let me know if it's ok now.